docs/topics/email.txt

```diff
@@ -745,6 +746,8 @@ a single call to send_messages::
 
 In this example, the call to ``send_messages()`` opens a connection on the
 backend, sends the list of messages, and then closes the connection again.
+(This connection overrides the :class:`EmailMessage.connection <EmailMessage>`
+property if it has been set on any of the individual messages.)
```

Comment:
I think instead of here, we can have this further up in the `Firstly, you can use the ``send_messages()`` method...` paragraph

```
Firstly, you can use the ``send_messages()`` method on a connection. This takes
a list of :class:`~django.core.mail.EmailMessage` instances (or subclasses),
and sends them all using that single connection. As a consequence, any
:class:`EmailMessage.connection <EmailMessage>` set on an individual message is
ignored.
```

Reply:
Done. I kept the "connection is ignored" in parens, as it's not essential to understanding send_messages().

(Also, since this file declares `.. module:: django.core.mail`, it's OK to omit that full module from Sphinx links. Unfortunately we can't link directly to <code>:attr:\`~EmailMessage.connection\`</code>, as the EmailMessage class docs don't use standard Sphinx roles to define attributes.)